torch_xla/csrc/token_handler.cpp

```diff
@@ -30,8 +31,19 @@ xla::XlaOp SliceOneToken(xla::XlaOp input) {
 
 }  // namespace
 
+#ifndef PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE
+#define PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE false
+#endif
+
 xla::XlaOp TokenHandler::GetInput(xla::XlaOp input,
                                   const xla::Shape* input_shape) {
+  static bool disable_numeric_token =
+      xla::sys_util::GetEnvBool("PT_XLA_DISABLE_NUMERIC_CC_TOKEN",
+                                PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE);
```

Comment:
I think it is OK to hard code `false` directly here. We generally don't use `ifdef` in pt/xla.

Reply:
OK, we'll append the following to the end of `torch_xla/__init__.py`:
```
_set_missing_env('DISABLE_NUMERIC_CC_TOKEN', 'true')
```